inference-engine/ie_bridges/python/tests/test_IECore.py

```diff
@@ -5,8 +5,9 @@
 import pytest
 from sys import platform
 from pathlib import Path
+import numpy as np
 
-from openvino.inference_engine import IENetwork, IECore, ExecutableNetwork
+from openvino.inference_engine import IENetwork, IECore, ExecutableNetwork, get_library_path
```

Comment:
do we need `get_library_path` in public API? I suppose IE core finds extensions located in the same folder by default, is not it?

Reply:
I checked, looks like it's really redundant